gnmi/config.go

```diff
@@ -23,6 +23,9 @@ type Config struct {
 	// Addr is  the listen address of the gNMI server.
 	Addr string `mapstructure:"addr"`
 
+	// CaFile is the CA certificate to use for mTLS.
+	CaFile string `mapstructure:"ca_file"`
```

Comment:
Should this be `CAFile` given that I think the "ca" here is for certificate authority?